experiments/azure/cyclecloud/cpu/experiment/data/amg2023/slurm-amg.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Should this be in configs with the others (why is amg2023 the only script under data)?

Reply:
They should be removed, and are now.